jenkins/scripts/select-compiler.sh

```diff
@@ -105,6 +105,22 @@ elif [ "$SELECT_ARCH" = "AIXPPC" ]; then
       ;;
   esac
 
+  case $NODE_NAME in
+    *aix71* )
```

Comment:
If it's too early to test this on AIX7.2, then this is fine, but I have reason to hope this will eventually be `*aix7* )`.

Reply:
Am I right in thinking we have a 7.2 machine in ci? if so this seperation is needed because it wont have yum installed gcc6

Reply:
we do not! :-) I setup 1 single 7.2 machine, but we can't add it until we have 3. And given what we now know about using yum to install gcc6, I'll want to revisit this. But you don't have to deal with this now. Forget I mentioned it, its unrelated to AIX7.1.